keanu-project/src/main/java/io/improbable/keanu/tensor/TensorShapeValidation.java

```diff
@@ -60,6 +60,18 @@ public static void checkDimensionExistsInShape(int dimension, long[] shape) {
         }
     }
 
+    public static void checkTensorsAreScalar(long[] ...shapes) {
```

Comment:
whitespace - should be `long[]... shapes`. If you format the file in IntelliJ it will make this change.

Reply:
This makes much more sense, `...` is really a type modifier

Reply:
fixed